CoCoding/apps/codes/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class CodesConfig(AppConfig):
+    name = 'codes'
```

Comment:
Oh my god.... hey dude, you should write app full python path............

Reply:
Oh thank you. You are so sweet guy